tests/testthat/_snaps/conventional-commits/NEWS.md

```diff
@@ -0,0 +1,40 @@
+<!-- NEWS.md is maintained by https://cynkra.github.io/fledge, do not edit -->
+
+## upkeep
+ update rlang usage.
+
+## fix
+ prevent racing of requests
+
+Introduce a request id and a reference to latest request. Dismiss
+incoming responses other than from latest request.
+
+Remove timeouts which were used to mitigate the racing issue but are
+obsolete now.
+
+Reviewed-by: Z
+Refs: #123
+## feat(lang)
+feat(lang): add Polish language
+
+## docs
+ correct spelling of CHANGELOG
+
+## chore!
+ drop support for Node 6
+
+BREAKING CHANGE: use JavaScript features not available in Node 6.
+
+## feat(api)!
+feat(api)!: send an email to the customer when a product is shipped
```

Comment:
I would have expected those to look a bit like the following, but let's tackle this in a future PR:

```suggestion
## docs

- correct spelling of CHANGELOG.

## chore

- drop support for Node 6.

## Breaking change

 - use JavaScript features not available in Node 6.

## Breaking change

### api

- send an email to the customer when a product is shipped

## feat

### api

- send an email to the customer when a product is shipped
```